draft-ietf-sframe-enc.md

```diff
@@ -475,38 +446,24 @@ header ----+------------------>| AAD
 +-----+                        |
    |                       AEAD.Encrypt
    |                           |
-   |                           V
-   |                   +-------+-------+
-   |                   |               |
-   |                   |               |
-   |                   |   encrypted   |
-   |                   |     frame     |
-   |                   |               |
-   |                   |               |
-   |                   +-------+-------+
-   |                           |
-   |                  generic RTP packetize
-   |                           |
-   |                           |
-   |    +----------------------+--------.....--------+
-   |    |                      |                     |
-   V    V                      V                     V
-+---------------+      +---------------+     +---------------+
-| SFrame header |      |               |     |               |
-+---------------+      |               |     |               |
-|               |      |  payload 2/N  | ... |  payload N/N  |
-|  payload 1/N  |      |               |     |               |
-|               |      |               |     |               |
-+---------------+      +---------------+     +---------------+
+   |     +---------------+     |
+   +---->| SFrame Header |     |
+         +---------------+     |
+         |               |     |
+         |               |<----+
+         |   ciphertext  |
+         |               |
+         |               |
+         +---------------+
 ~~~~~
-{: title="Encryption flow with per-frame encryption" }
+{: title="Encryption flow" }
 
 ### Decryption
 
 Before decrypting, a client needs to assemble a full SFrame ciphertext.  When
 SFrame is applied per-packet, this is done by extracting the payload of a
 decrypted SRTP packet.  When SFrame is applied per-frame, the receiving client
```

Comment:
References here to SRTP and RTP should probably also be removed/moved to examples section.

Reply:
Suggested rewrite to make this RTP-neutral:

> Before decrypting, a client needs to assemble a full SFrame ciphertext.  When an SFrame ciphertext may be fragmented into multiple parts for transport (e.g., a whole encrypted frame sent in multiple SRTP packets), the receiving client collects all the fragments of the ciphertext, using an appropriate sequencing and start/end markers in the transport.  Once all of the required fragments are available, the client reassembles them into the SFrame ciphertext, then passes the ciphertext to SFrame for decryption.

That also highlights another thing that the application needs to do (for #103), namely provide the appropriate fragmentation and reassembly layer if you're not going to align SFrame ciphertext to transport units.

Reply:
https://github.com/sframe-wg/sframe/pull/115/commits/094a8b57ddecf38d7601a832f77aaf2dbc9cb280

Should address the latter.